opening_hours/tests/test_reservable_time_spans_client.py

```diff
@@ -0,0 +1,707 @@
+import datetime
+from zoneinfo import ZoneInfo
+
+import pytest
+from django.utils import timezone
+from django.utils.timezone import get_default_timezone
+from freezegun import freeze_time
+
+from opening_hours.enums import State
+from opening_hours.errors import ReservableTimeSpanClientNothingToDoError
+from opening_hours.models import ReservableTimeSpan
+from opening_hours.utils.hauki_api_client import HaukiAPIClient
+from opening_hours.utils.hauki_api_types import (
+    HaukiAPIOpeningHoursResponseDate,
+    HaukiAPIOpeningHoursResponseItem,
+    HaukiAPIOpeningHoursResponseResource,
+    HaukiAPIOpeningHoursResponseTime,
+    HaukiTranslatedField,
+)
+from opening_hours.utils.reservable_time_span_client import ReservableTimeSpanClient, TimeSpanElement
+from tests.decorators import patch_method
+
+# Applied to all tests
+pytestmark = [
+    pytest.mark.django_db,
+]
+
+DEFAULT_TIMEZONE = get_default_timezone()
+
+
+def _get_date(*, day: int = 1, hour: int = 0, tzinfo: ZoneInfo | None = None):
+    return datetime.datetime(2023, 1, day, hour, 0, 0, tzinfo=tzinfo or DEFAULT_TIMEZONE)
+
+
+def _get_resource_opening_hours() -> HaukiAPIOpeningHoursResponseItem:
+    """
+    Contains cases of opening hours, to test the parsing logic.
+    Day 1:
+        10:00 - 16:00 (with reservation)
+        10:00 - 12:00 (open and reservable)
+        14:00 - 16:00 (closed)
+    Day 2:
+        10:00 - 20:00 (undefined)
+        18:00 - 06:00 (open and reservable)
+
+    Day 1 is reservable from 10:00 to 12:00 and from 14:00 to 16:00
```

Comment:
I think the logic here is correct but is this specific comment invalid or am I misinterpreting it? Should it read (14-16.00 of day is closed): 
Day 1 is reservable from **10:00 to 14:00**
Day 2 is reservable from **20:00 to 06:00**

Reply:
Good catch, yes looks like the comment part is incorrect.